infra/recipes/k8s/environments/azure/sandbox/tables/values.yaml

```diff
@@ -0,0 +1,34 @@
+namespace: "default"
+
+openhouseClusterTag: "openhouse"
+
+clusterPropertiesPath: "config/azure/sandbox/cluster.yaml"
+
+tablesService:
+  replicaCount: 1
+  image:
+    repository: "containerregistryopenhouse.azurecr.io/openhouse-tables-service"
+    tag: "latest"
+    pullPolicy: IfNotPresent
+  service:
+    type: NodePort
```

Comment:
Should this be ClusterIP?

Reply:
Yes I will change back, I was just experimenting with it